BeardedSpice/MediaStrategies/Spotify.js

```diff
@@ -16,11 +16,11 @@ BSStrategy = {
     args: ["URL"]
   },
   isPlaying: function() {
-    return !!(document.querySelector('div.player-controls__buttons button[data-testid="control-button-pause"]'));
+    return !!(document.querySelector('div.player-controls__buttons button[data-testid="control-button-playpause"]'));
```

Comment:
Why here 3 selectors are equal?

Reply:
It looks like the data_testid is now the same for Play and Pause. "control-button-pause" & "control-button-play" doesn't seems to exist anymore.

![image](https://user-images.githubusercontent.com/19412894/191123044-3e8e0c79-4243-4f12-b2e0-b3b9efd7a3dc.png)

Reply:
I understand that, but my question was deeper...
1. That is, now there is no way to understand when the player is playing? So the `isPlaying()` function is not possible to implement.
2. Why in the `toggle()` function does two identical selectors through the `OR` operator?